action/src/utils.ts

```diff
@@ -167,4 +174,92 @@ export class Utils {
     public static isWindows() {
         return platform().startsWith('win');
     }
+    public static async validatePlatfornUrl(jfrogUrl: string): Promise<void> {
+        const jfrogUrlFailure: string = 'JF_URL must point on your full platform URL, for example: https://mycompany.jfrog.io/, make sure the platform is up and running and accessible.'
+        //verify that the provided JFrog URL is valid and responsive
+        const pingUrl: string = jfrogUrl!.replace(/\/$/, '') + '/artifactory/api/system/ping';
+        const httpClient: HttpClient = new HttpClient();
+        let response: HttpClientResponse
+        try {
+             response = await httpClient.get(pingUrl);
+        }catch (error: any) {
```

Comment:
Please make sure to run `npm run format` to ensure no formatting issues
```suggestion
        } catch (error: any) {
```